pkg/builders/s2i/builder_test.go

```diff
@@ -150,6 +150,67 @@ func Test_BuilderImageConfigurable(t *testing.T) {
 	}
 }
 
+// Test_BuildImageWithFuncIgnore ensures that ignored files are not added to the func
+// image
+func Test_BuildImageWithFuncIgnore(t *testing.T) {
+
+	funcIgnoreContent := []byte("hello.txt")
+	f := fn.Function{
+		Runtime: "node",
+		Root:    ".",
```

Comment:
You can use something like `filepath.Join(t.TempDir(), "fn")` as `Root`. Then directory removal will done automatically no need for defers.